src/shared/contexts/editor/index.tsx

```diff
@@ -65,7 +65,11 @@ export const EditorProvider: FC = ({children}) => {
 
   const injectViaLsp = useCallback(
     (cmd, data: Omit<ExecuteCommandArgument, 'textDocument'>) => {
-      connection.current.inject(cmd, data)
+      try {
+        connection.current.inject(cmd, data)
+      } catch (e) {
+        console.error(e)
+      }
```

Comment:
The monitor-ci-test kept failing saying `Cannot read properties of null (reading 'current')`. Here is an example of a failed test https://app.circleci.com/pipelines/github/influxdata/monitor-ci/12979

My guess why it fails is that cypress runs test commands so quick that before the monaco editor is mounted, `injectViaLsp` is called by selecting a measurement. Adding an assertion on monaco editor (with test ID `flux-editor`) as well as a try catch logic here resolve the problem